test/hutch.test.js

```diff
@@ -83,6 +83,55 @@ describe('Hutch', function() {
     });
   });
 
+  it('should publish a message using the same channel', function(complete) {
+
+    var hutch = new AMQPHutch();
+
+    hutch.initialise({
+      connectionString: 'amqp://localhost',
+      retryWait:        100
+    });
+
+    hutch.on('ready', function() {
+
+      var options = {
+        exchange: {
+          name: 'example.exchange.1',
+          type: 'topic'
+        },
+        queue: {
+          name: 'example.queue',
+          prefetch: 1,
+          durable:  true
+        },
+        publish: {
+          persistent: true,
+          expiration: 86400000
+        }
+      };
+
+      var count = 0;
+      var consumer = function(message, done, fail) {
+        count++;
+        done();
+
+        if(count === 2){
+          var channel = hutch._getChannelByExchange(options.exchange.name);
+          channel.should.exist();
```

Comment:
It doesn't, but it does check to see if we can pull out the channel by exchange which is what happens prior to the publish call